cs/benchmark/SecondaryIndexes.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using FASTER.core;
+
+namespace FASTER.benchmark
+{
+    class NullKeyIndex<Key> : ISecondaryKeyIndex<Key>
+    {
+        public string Name => "KeyIndex";
+
+        public bool IsMutable => true;
+
+        public void Delete(ref Key key) { }
+
+        public void Insert(ref Key key) { }
```

Comment:
There is a variant here that takes in the address as well. In that case, we need to update the index whenever addresses changes, even if the key does not change. Like an Upsert call here in addition to Insert.